javascript/webpacker_react-npm-module/package.json

```diff
@@ -13,5 +13,8 @@
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-react": "^6.10.0"
+  },
+  "dependencies": {
+    "lodash": "^4.0.0"
```

Comment:
Currently we use 4 functions, where `keys()` and `assign()` would require polyfill (as far as I understand) anyway and `intersection()` and `omit()` may require own implementation. I am happy to explore this as soon as we get the basic functionalities in place, so we can promote the gem and spread the word.